js-envs/src/main/scala/org/scalajs/jsenv/rhino/LazyScalaJSScope.scala

```diff
@@ -56,18 +62,23 @@ class LazyScalaJSScope(
   override def getClassName() = "LazyScalaJSScope"
 
   override def get(name: String, start: Scriptable) = {
-    fields.getOrElse(name, {
-      try {
-        load(name)
-        fields.getOrElse(name, Scriptable.NOT_FOUND)
-      } catch {
-        // We need to re-throw the exception if `load` fails, otherwise the
-        // JavaScript runtime will not catch it.
-        case t: ScalaJSCoreLib.ClassNotFoundException =>
-          throw Context.throwAsScriptRuntimeEx(t)
-      }
-    }).asInstanceOf[AnyRef]
+    if (name == "__noSuchMethod__") {
```

Comment:
I see. Could you drop a comment about that, please? I it seems to be a pretty obscure Rhino feature